### PR TITLE
fix: [Noticket] fix file read

### PR DESCRIPTION
**Pull request by zhutoutou**
There are two problem here:
- I get file failed with FileNotFoundError in unittest
- we need read the file content.

### CODE REVIEW

**Review by dixon629 — APPROVED** [review]

File: llm/llmapi.py [hunk 1]
```diff
@@ -25,8 +25,10 @@ def summarize_pr_info(info) -> str:
 
 
 def _generate_summary_messages(info: PRInfo) -> List[Message]:
-    message = Message(Role.USER,
-                      f"Can you summarize the PR based on the following info?\nthe title is: {info.title}\nand the description is: {info.description}")
+    message = Message(
+        Role.USER,
+        f"Can you summarize the PR based on the following info?\nthe title is: {info.title}\nand the description is: {info.description}",
```
Comment (Br1ze) [inline llm/llmapi.py:30]: Test Comment by line

Comment (Br1ze) [inline llm/llmapi.py:30]: Test Comment by line
